src/App.js

```diff
@@ -1,30 +1,22 @@
-import Stack from "@mui/material/Stack";
-import Button from "@mui/material/Button";
-import logo from "./logo.svg";
+import React from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Home from "./pages/Home.js";
+import Menu from "./pages/Menu.js";
+import Contact from "./pages/Contact.js";
+import Navbar from "./components/Navbar.js";
 import "./App.css";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-        <Stack spacing={2} direction="row">
-          <Button variant="text">Text</Button>
-          <Button variant="contained">Contained</Button>
-          <Button variant="outlined">Outlined</Button>
-        </Stack>
-      </header>
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
```

Comment:
I just realized there would be a difference between a landing page and the home page or not.

if that's the case then we home page link can go away as the donut shop logo serves as the home and we need to add `exact` prop so the home doesn't load accidentally on other pages, unless react-router has updated this behavior 

As you can see clicking home, binds home to URL 
![image](https://github.com/Muniba-Ahmed/donutshop/assets/50847476/af62aaa5-e620-4f02-851a-9bb0c76d380a)

In addition, clicking other nav buttons, URL should update to page route but it should not be capital, everything is lowercase

Reply:
I've removed the home button and now the logo is routing to the home/landing page.